imports/documentGeneration.js

```diff
@@ -0,0 +1,119 @@
+import { Meteor } from 'meteor/meteor';
+
+import { Minutes } from './minutes';
+import { Topic } from './topic';
+import { Attachment } from './attachment';
+import { GlobalSettings } from './config/GlobalSettings';
+import { InfoItemFactory } from './InfoItemFactory';
+
+import './helpers/promisedMethods';
+
+export class DocumentGeneration {
+    // ********** static methods ****************
+    static async downloadMinuteProtocol(minuteID) {
+        //Create HTML
+        let htmldata = await Meteor.callPromise('documentgeneration.createHTML', minuteID);
+        let currentMinute = new Minutes(minuteID);
+        //Download File
+        let fileBlob = new Blob([htmldata], {type:'octet/stream'});
+
+        const filename = currentMinute.parentMeetingSeries().getRecord().name + '-' + Minutes.findOne(minuteID).date + '.html';
+
+        if (window.navigator.msSaveOrOpenBlob)  // necessary for Internet Explorer, since it does'nt support the download-attribute for anchors
```

Comment:
Please consider putting curly braces around this case. The fact that they are missing is easily missed and if someone extends this case in the future weird things may happen.